client/components/place/place-list/place-list.component.js

```diff
@@ -8,10 +8,14 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         var ctrl = this;
         var places = [];
         var tracks = [];
-        var placeCounter;
+        var placeCounter = 1;
         var trackCounter;
         var placeTypeLength;
         var trackTypeLength;
+        var key;
+        var map;
+
+        var realPlacesType = [];
```

Comment:
I don't know how I can it rename correctly. Maybe you adviced me?
It's array, which contain only name of types, which are checked on filter.